lib/features/thread/presentation/thread_controller.dart

```diff
@@ -449,6 +449,7 @@ class ThreadController extends BaseController with EmailActionController {
       searchController.searchEmailFilter.value = _searchEmailFilter.clearBeforeDate();
       _searchEmail(limit: limitEmailFetched);
     } else {
+      await _cachingManager.cleanEmailCache();
```

Comment:
what is the problem here? why need to clean the cache here?

Reply:
If I don't clean the cache when the list email has one element. it will call back the cache and yield that cache

Pls watch my demo


https://user-images.githubusercontent.com/99852347/219533434-7b7a777a-429f-4368-830e-68b47db5a2f1.mov